crates/wysiwyg/src/composer_model/lists.rs

```diff
@@ -253,9 +254,11 @@ where
                 list_type,
                 vec![DomNode::Container(ContainerNode::new_list_item(
                     "li".into(),
-                    vec![DomNode::new_text(S::default())],
+                    vec![DomNode::new_text("\u{200b}".into())],
```

Comment:
It would be nice if we could create a constant for this somewhere, but we'd have to have one for String, another one for UnicodeString... maybe it's not worth it.